src/IB/IBImplicitStaggeredHierarchyIntegrator.cpp

```diff
@@ -108,6 +109,18 @@ namespace
 {
 // Version of IBImplicitStaggeredHierarchyIntegrator restart file data.
 static const int IB_IMPLICIT_STAGGERED_HIERARCHY_INTEGRATOR_VERSION = 1;
+
+static void ib_4_interp_fcn(const double r, double* const w)
```

Comment:
(1) I don't want to add the entire header file just for ib_4 function definition
(2) We might be testing different delta functions with preconditioners and this seems to be the right place for their definition...